### PR TITLE
Modified Classes to conform to plaintext style guides

### PR DESCRIPTION
**Pull request by bleehu**
Isn't that easier to read? It will also be easier for CXDocs to read...

### CODE REVIEW

File: CharacterCreation/02_Classes.txt [hunk 1]
```diff
@@ -1,25 +1,44 @@
...
+* Submachine Guns 
+* Shotguns 
+* Carbines
+* Assault Rifles
+* Battle Rifles.
+
+Armor Proficiencies: 
+* Light
+* Medium
+* Medium-Heavy
```
Comment (1sourcecontrol) [inline CharacterCreation/02_Classes.txt:39]: Idk. 16 lines to display 3 lines of information. I really don't like all the space; I think it will make it much harder to scroll through the doc to compare classes.

Comment (bleehu) [reply CharacterCreation/02_Classes.txt:39]: 1) This is a public repo, get my name out of there. Get it out now. 
2) If you want to address me, @ me. 
3) I hear you, long files are bad. But by breaking up the information, it actually becomes easier to skim to the part of the info that you're looking for thereby making it easier to find and work on what you're trying to do. For players, they'll be using the CXDocs page which will render each of these into Divs which can then be aligned to be side-by-side. Better yet, if the computer can read and differentiate each class, we may be able to actually add a side-by-side tool. But in order to add that quality of life, we need CXDocs to be able to read what we're writing rather than needing a human-in-the-loop to interpret each change.

Comment (Turtlelord26) [reply CharacterCreation/02_Classes.txt:39]: I concur that spacing the information vertically will make comparison faster despite marginally increased scrolling time.

**Comment by bleehu** [comment]
You would not *BELIEVE* how much trouble line 216 with the non-ascii backtick character is giving me. Came from copy-pasting from Microsoft word, which uses Windows-1252 standard language encoding. God knows it's not the last time it's gonna happen. so I've gotta figure out a way to future proof it.

File: CharacterCreation/02_Classes.txt [hunk 2]
```diff
@@ -512,15 +710,28 @@ Class Feats:
...
+
 Class Primary Attribute: Strength
-Passives: Expert Unarmed
-Weapon Proficiencies: Pistols.
-Armor Proficiencies: Concealed, Recon, Light, Medium, Medium-Heavy.
 Starting Wealth: $2170 + 100*Luck
-Class Feats:
+
+Passives:
+* Gain Expert in Unarmed
```
Comment (1sourcecontrol) [inline CharacterCreation/02_Classes.txt]: Can you change this to something like , "Gain usage of melee weapon "Hand-to-Hand [Expert]", disregarding requirement"?

I don't think there is an "Expert in Unarmed" and I believe "Unarmed Expert" was changed.

(If this isn't too much scope creep)

**Review by 1sourcecontrol — APPROVED** [review]
One potential change, go ahead and merge when you're done

**Comment by 1sourcecontrol** [comment]
Looks good though! Nice work.

File: CharacterCreation/02_Classes.txt (hunk 2 above)
Comment (bleehu) [reply CharacterCreation/02_Classes.txt]: wilco
